.agents/skills/typescript-expert/scripts/ts_diagnostic.py

```
@@ -0,0 +1,225 @@
+        errors = result.count("error TS")
+        print(f"  ❌ {errors}+ type errors found")
+        print(result[:500])
+    else:
+        print("  ✅ No type errors")
+
+def check_any_usage(project_root: str = "."):
+    """Check for any type usage."""
+    print("\n⚠️ 'any' Type Usage:")
+    print("-" * 40)
+    
+    src_path = os.path.join(project_root, "src") if project_root != "." else "src/"
```

Ruff (0.15.1)

[error] Undefined name `os`

(F821)